azext_iot/_params.py

```diff
@@ -495,6 +497,37 @@ def load_arguments(self, _):
             "Account and Contributor role for the IoT Hub.",
         )
 
+    with self.argument_context("iot hub device-identity generate-mqtt-credentials") as context:
+        context.argument(
+            "connection_string",
+            options_list=["--connection-string", "--cs"],
+            help="Target connection string. This bypasses the IoT Hub registry and generates the user credentials directly "
+                 "from the supplied symmetric key without further validation. All other command parameters aside from "
+                 "duration will be ignored. Supported connection string types: Device, Module."
+        )
+        context.argument(
+            "duration",
+            options_list=["--duration", "--du"],
+            type=int,
+            help="Indicates the duration in seconds, for which the mqtt credentials will remain valid.",
+        )
+        context.argument(
+            "product_info",
+            options_list=["--product-info", "--pi"],
+            help="Description of the product.",
+        )
+        context.argument(
+            "dtmi",
+            options_list=["--dtmi"],
+            help="Digital Twins model Id. Example: dtmi:com:example:Room;2",
+        )
+        context.argument(
+            "version",
+            arg_type=get_enum_type(MQTTConnectVersionType),
+            options_list=["--format-version", "--fv"],
+            help="MQTT connect credentials format version to use when generating the credentials.",
```

Comment:
The help should state when to use which format i.e. api versions.

Reply:
Something like:
Use v1 for MQTT 3.1 or below and v2 for MQTT 3.1.1 or higher